### ISSUE TITLE
attention_ocr: run the whole test dataset

### ISSUE THREAD

**Issue by JingLiJJ** [tensorflow/models#5813]
System information
What is the top-level directory of the model you are using: attention_ocr
TensorFlow version (use command below): 1.8.0
CUDA/cuDNN version: 7.1.2

Describe the problem
Thanks a lot for your work, I have a question about eval.py. 
I know that you use 100*32 images to test model randomly. Now, I want to test model on the whole test datasets. So, I change tf.train.shuffle_batch to tf.train.batch in data_provider.py. Now, the data would be chosen in order. Then, I set num_batches to  dataset_size/batch_size. Am I right? Can the model test the whole test dataset now?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
Bazel version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by R1234A** [comment]
@JingLiJJ Have you got the answer as I have trained the model but using demo_inference.py is not giving accurate results. So should I use eval.py for results??
